scripts/install.sh

```diff
@@ -7,110 +7,106 @@ DONE_START='\n\e[1;32m' # new line + bold + green
 do
    case "$opt" in
-      c ) config_dir_path="$OPTARG" ;;
       n ) network="$OPTARG" ;;
-      m ) contracts_only=true ;;
+      d ) skip_deployment=${OPTARG:-true} ;;
+      b ) skip_client_build=${OPTARG:-true} ;;
       h ) help ;;
       ? ) help ;; # Print help in case parameter is non-existent
    esac
 done
 shift $(expr $OPTIND - 1) # remove options from positional parameters
 
 # Overwrite default properties
-CONFIG_DIR_PATH=${config_dir_path:-$CONFIG_DIR_PATH_DEFAULT}
 NETWORK=${network:-$NETWORK_DEFAULT}
-CONTRACTS_ONLY=${contracts_only:-false}
+SKIP_DEPLOYMENT=${skip_deployment:-false}
+SKIP_CLIENT_BUILD=${skip_client_build:-false}
 
 # Run script
 printf "${LOG_START}Starting installation...${LOG_END}"
 
-printf "Config dir path: $CONFIG_DIR_PATH\n"
-printf "Network: $NETWORK"
+printf "Network: $NETWORK\n"
 
-cd $KEEP_CORE_SOL_PATH
+cd $KEEP_BEACON_SOL_PATH
 
-printf "${LOG_START}Installing NPM dependencies...${LOG_END}"
-npm install
+printf "${LOG_START}Installing YARN dependencies...${LOG_END}"
+# rm -rf node_modules/
+yarn install
 
-if [ "$NETWORK" == "local" ]; then
+if [ "$NETWORK" == "development" ]; then
     printf "${LOG_START}Unlocking ethereum accounts...${LOG_END}"
     KEEP_ETHEREUM_PASSWORD=$KEEP_ETHEREUM_PASSWORD \
-        npx truffle exec scripts/unlock-eth-accounts.js --network $NETWORK
+        npx hardhat unlock-accounts --network $NETWORK
 fi
 
-printf "${LOG_START}Migrating contracts...${LOG_END}"
-rm -rf build/
+printf "${LOG_START}Building contracts...${LOG_END}"
 
-CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=$CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY \
-    npx truffle migrate --reset --network $NETWORK
+rm -rf build && rm -rf cache && rm -rf typechain
+yarn build
 
-printf "${LOG_START}Copying contract artifacts...${LOG_END}"
-rm -rf artifacts
-cp -r build/contracts artifacts
-npm link
+if [ "$SKIP_DEPLOYMENT" = false ] ; then
+   printf "${LOG_START}Deploying contracts...${LOG_END}"
+   # rm -rf deployments/development/
 
-printf "${LOG_START}Initializing contracts...${LOG_END}"
+   CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=$CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY \
+      npx hardhat deploy --reset --export export.json --network $NETWORK
+fi
+    
+printf "${LOG_START}Setting up staking...${LOG_END}"
 
 CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=$CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY \
-    npx truffle exec scripts/delegate-tokens.js --network $NETWORK
-
-if [ "$CONTRACTS_ONLY" = false ] ; then
-  printf "${LOG_START}Updating keep-core client configs...${LOG_END}"
-  for CONFIG_FILE in $CONFIG_DIR_PATH/*.toml
-  do
-      KEEP_CORE_CONFIG_FILE_PATH=$CONFIG_FILE \
-      CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=$CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY \
-          npx truffle exec scripts/lcl-client-config.js --network $NETWORK
-  done
-
-  printf "${LOG_START}Building keep-core client...${LOG_END}"
-  cd $KEEP_CORE_PATH
-  go generate ./...
-  go build -a -o keep-core .
+    npx hardhat staking --network $NETWORK
```

Comment:
Does it mean only one client is supported? We need more - with one client we cannot test network and anything that depends on at least two unique operators participating in the protocol (sortition pool selection, ECDSA key generation retries, ECDSA signing retries, broadcast channel filters, and more).